UPGRADE-3.1.md

```diff
@@ -103,11 +103,10 @@ FrameworkBundle
 
    ```yaml
    framework:
-       serializer: ~
```

Comment:
The doc states that enabling should be done explicitly using `enabled: true`:
http://symfony.com/doc/current/cookbook/serializer.html
That's why I made this change: `serializer: ~` could be misleading also.
Not adding it here will force people using implicit enabling to think about it, which is a good thing to me :)